ee/se/modules/380-metallb/images/artifact/werf.inc.yaml

```diff
@@ -1,23 +1,34 @@
 ---
+artifact: {{ .ModuleName }}/metallb-src-artifact
+fromArtifact: common/src-artifact
+git:
+- add: /{{ .ModulePath }}modules/380-{{ .ModuleName }}/images/artifact/patches
+  to: /patches
+  stageDependencies:
+    install:
+    - '**/*'
+shell:
+  install:
+  - git clone -b v0.14.8 {{ $.SOURCE_REPO }}/metallb/metallb.git /src
+  - cd /src
+  - git apply /patches/*.patch --verbose
```

Comment:
please add `rm -rf .git`

Reply:
removed git clone from the shell section, the .git folder is no longer there, no need to delete